src/index.ts

```diff
@@ -22,10 +19,11 @@ const fetchJsonFromUrl = async (url: string): Promise<JsonObject> => {
 // random string identifier
 const getRandomString = () => Math.random().toString(36).substr(2, 9)
 
-// given a json object and result array, collect all jpg image url in result array
+// given a json object and result array, collect all jpg/jpeg/png image url in result array
```

Comment:
If you really want to add flexibility gifs should be here as well (because of possible first aid animations in the future. Not something that we want to deal with now too)